src/Microsoft.NetCore.Analyzers/Core/Security/DoNotSetSwitch.cs

```diff
@@ -27,18 +33,38 @@ public sealed class DoNotDisableSchUseStrongCrypto : DiagnosticAnalyzer
             SystemSecurityCryptographyResources.ResourceManager,
             typeof(SystemSecurityCryptographyResources));
 
-        internal static DiagnosticDescriptor Rule = new DiagnosticDescriptor(
-                DiagnosticId,
-                s_Title,
-                s_Message,
-                DiagnosticCategory.Security,
-                DiagnosticHelpers.DefaultDiagnosticSeverity,
-                isEnabledByDefault: DiagnosticHelpers.EnabledByDefaultIfNotBuildingVSIX,
-                description: s_Description,
-                helpLinkUri: null,
-                customTags: WellKnownDiagnosticTags.Telemetry);
+        internal static DiagnosticDescriptor DoNotDisableSchUseStrongCryptoRule = new DiagnosticDescriptor(
+            DiagnosticId,
+            s_Title,
+            s_Message,
+            DiagnosticCategory.Security,
+            DiagnosticHelpers.DefaultDiagnosticSeverity,
+            isEnabledByDefault: DiagnosticHelpers.EnabledByDefaultIfNotBuildingVSIX,
+            description: s_Description,
+            helpLinkUri: null,
+            customTags: WellKnownDiagnosticTags.Telemetry);
+        internal static DiagnosticDescriptor DoNotDisableSpmSecurityProtocolsRule = SecurityHelpers.CreateDiagnosticDescriptor(
+            "CA5378",
```

Comment:
Nit: Can we be consistent in use of fields vs inlining the argument values between the two descriptors?